rest_framework/pagination.py

```diff
@@ -198,6 +198,10 @@ def paginate_queryset(self, queryset, request, view=None):
             return None
 
         paginator = self.django_paginator_class(queryset, page_size)
+
+        # paginator needed in get_page_number()
+        # Regression in 6f6d402d04 and/or c05cbe2da213/
+        self.paginator = paginator
```

Comment:
Hey @tomchristie — there's a regression in in 6f6d402d04 and/or c05cbe2da213 — I've just put this here to see what the CI says (since I'm trying to update for the s/master/main/ on Django) but we'll need to pull this out and fix it properly.

Reply:
Thanks. Don't think there has *ever* been a point in history where a change has been applied without a PR that *hasn't* quietly broken things. Still, sorted now... https://github.com/encode/django-rest-framework/commit/39a98c80a6cb0b21fb6adb5ee97bc64b8b404433

Still looks like there's a failing test on `tests/test_description.py:170: in test_markdown` that's non-obvious to me.

Reply:
Let's drop the `self.paginator = ...` changes in the PR.

Reply:
> Still looks like there's a failing test on tests/test_description.py:170: in test_markdown that's non-obvious to me.

Did you `pip install -U markdown`? 

Will rebase now.

Reply:
Gotcha, that sorted it.